src/interface/meta.thrift

```diff
@@ -145,26 +129,6 @@ struct HostItem {
     4: map<string, list<common.PartitionID>> (cpp.template = "std::unordered_map") all_parts,
 }
 
-struct UserItem {
```

Comment:
We don't need it any more?

Reply:
I don't think we need it any more, currently we only have one item of password, we can storage it direct.
If there are new elements in the future, we can add this struct at that time.